search-app/src/cmr/search/system.clj

```diff
@@ -164,20 +165,16 @@
   "Performs side effects to initialize the system, acquire resources,
   and start it running. Returns an updated instance of the system."
   [this]
-  (info "search System starting")
```

Comment:
Why are you removing these start/stop info log events?

Reply:
common-lib/src/cmr/common/system.clj should handle this